fix(guide): add enable advanced settings to to Recommended naming scheme (#1705)
TRaSH-Guides/Guides: docs/Radarr/Radarr-recommended-naming-scheme.md

------

## Preparation

> Go to `Settings` => `Media Management` and make sure you have enabled `Show Advanced` on the top.

Reviewer: ```suggestion
Go to `Settings` => `Media Management` and make sure that `Show Advanced` at the top is enabled.
```

![Enable Advanced](images/radarr-show-adavanced.png)

> and change it to enable the advanced view.

Reviewer: ```suggestion
Once you clicked on the button it should look like this and you should see all the advanced options.
```
